### DISCUSSION TITLE
Scrape historical data using custom asset

### DISCUSSION THREAD

**Discussion by Marketos-Damigos** [ghostfolio/ghostfolio#3002] [Q&A]
Hello,

Is it possible to use the manual data source and the scraper to get historical data from an HTML table?

An example would be the following: [https://markets.ft.com/data/funds/tearsheet/historical?s=SE0005188836:SEK](https://markets.ft.com/data/funds/tearsheet/historical?s=SE0005188836:SEK)

**Comment by dtslvr** [comment]
Hi @Marketos-Damigos

No, the scraper is specifically designed for daily fetching of the closing market price. For historical data, I recommend utilizing the import functionality instead.
